clang/test/CIR/CodeGen/bitfield-ops.c

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir -mmlir --mlir-print-ir-before=cir-lowering-prepare %s -o %t.cir 2>&1 | FileCheck %s -check-prefix=BEFORE
+
+typedef struct {
+  int a : 4;
+  int b : 27;
+  int c : 17;
+  int d : 2;
+  int e : 15;
+  unsigned f; 
+} S; 
+
+// BEFORE: cir.func {{.*@store_field}}
+// BEFORE:   [[TMP0:%.*]] = cir.alloca !ty_22S22, cir.ptr <!ty_22S22>, ["s"] 
+// BEFORE:   [[TMP1:%.*]] = cir.const(#cir.int<3> : !s32i) : !s32i
+// BEFORE:   [[TMP2:%.*]] = cir.get_member [[TMP0]][2] {name = "e"} : !cir.ptr<!ty_22S22> -> !cir.ptr<!u16i>
+// BEFORE:   [[TMP3:%.*]] = cir.set_bitfield [[TMP1]], [[TMP2]](#e) : (!s32i, !cir.ptr<!u16i>) -> !s32i
```

Comment:
Please also add a check line for the definition of `#e`, same for `#d` in the next example.